spec/xclarity_client_storage_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'spec_helper'
+
+describe XClarityClient do
+  before :all do
+    WebMock.allow_net_connect! #-- Uncomment this line if you're testing with a external mock.
+  end
+
+  let(:conf) do
+    XClarityClient::Configuration.new(
+      username:   ENV['LXCA_USERNAME'],
+      password:   ENV['LXCA_PASSWORD'],
+      host:       ENV['LXCA_HOST'],
+      port:       ENV['LXCA_PORT'],
+      auth_type:  ENV['LXCA_AUTH_TYPE'],
+      verify_ssl: ENV['LXCA_VERIFY_SSL']
+    )
+  end
+
+  let(:client) { XClarityClient::Client.new conf }
+
+  it 'has a version number' do
+    expect(XClarityClient::VERSION).not_to be nil
+  end
+
+  context 'with discover_storages' do
+    before do
+      @storages = client.discover_storages
+    end
+
+    describe 'GET /storage' do
+      it 'should respond with an array' do
+        expect(@storages.class).to eq(Array)
+        expect(@storages.any?).to eq(true)
+      end
+
+      it 'correct response values' do
+        expect(@storages[0].name).to eq('S3200-1')
```

Comment:
__Required:__
Please, store the `@storages[0]` in a variable and test against it.

This will provide a better maintenance, where if the tested index change, you only need to change it in only one place.